server_admin/topics/admin-console.adoc

```diff
@@ -7,12 +7,19 @@ include::realms/proc-using-admin-console.adoc[leveloffset=1]
 include::realms/master.adoc[leveloffset=2]
 include::realms/proc-creating-a-realm.adoc[leveloffset=2]
 include::realms/ssl.adoc[leveloffset=2][]
-include::realms/cache.adoc[leveloffset=2][]
 include::realms/email.adoc[leveloffset=2]
 include::realms/themes.adoc[leveloffset=2]
+include::realms/keys.adoc[leveloffset=2]
+include::realms/events.adoc[leveloffset=2]
+include::realms/internationalization.adoc[leveloffset=2]
+include::realms/security-defenses.adoc[leveloffset=2]
+include::realms/sessions.adoc[leveloffset=2]
+include::realms/tokens.adoc[leveloffset=2]
+include::realms/client-policies.adoc[leveloffset=2]
+include::realms/user-profile.adoc[leveloffset=2]
+include::realms/user-registration.adoc[leveloffset=2]
 include::login-settings.adoc[leveloffset=2]
 include::login-settings/forgot-password.adoc[leveloffset=2]
 include::login-settings/remember-me.adoc[leveloffset=2]
 include::login-settings/acr-to-loa-mapping.adoc[leveloffset=2]
-include::login-settings/update-email-workflow.adoc[leveloffset=2]
-include::realms/keys.adoc[]
+include::login-settings/update-email-workflow.adoc[]
```

Comment:
@agagancarczyk Any changes to the sections include will need to be enclosed in conditional code.
See https://docs.google.com/document/d/1VnBqy2WivM0VIgaz54sIJM0OQZH0krhhiylrYPg10Hg/edit#heading=h.ksbrxfbdrac1

Also, for this release we are reusing the same sections, because other chapters already cover these topics. For example, the User Profile section is already documented in the Managing Users chapter: https://www.keycloak.org/docs/latest/server_admin/index.html#user-profile

Reply:
Thanks Andy, I have cleaned this up as instructed.